configs/projects/agent-runtime.rb

```diff
@@ -0,0 +1,69 @@
+project 'agent-runtime' do |proj|
+  platform = proj.get_platform
+  # TODO: Is this an acceptable way to inherit from multiple possible agent version branches?
+  proj.setting(:agent_branch, ENV['AGENT_BRANCH'] || 'master')
+
+  proj.inherit_settings 'puppet-agent', 'git://github.com/puppetlabs/puppet-agent', proj.settings[:agent_branch]
+  # Set the version based on `git describe`, then prepend the branch name
+  # TODO: (It seems like there must be a better way to name the package - Is this valuable?)
+  proj.version_from_git
+  proj.version "#{proj.settings[:agent_branch]}.#{proj.settings[:package_version]}"
+  proj.generate_archives true
+  proj.generate_packages false
+
+  proj.description "The puppet agent runtime contains third-party components needed for the puppet agent"
+  proj.license "See components"
+  proj.vendor "Puppet, Inc.  <info@puppet.com>"
+  proj.homepage "https://puppet.com"
+
+  proj.setting(:artifactory_url, "https://artifactory.delivery.puppetlabs.net/artifactory")
+  proj.setting(:buildsources_url, "#{proj.artifactory_url}/generic/buildsources")
+
+  # At least on el-7, ruby_bindir should be set to /opt/puppetlabs/puppet/bin/, which is the same as the regular bindir:
+  # This does not seem to arrive here from the agent project's settings.
+  proj.setting(:ruby_bindir, "#{proj.settings[:bindir]}")
```

Comment:
fyi, it looks like this is only set for windows in the agent config, so that's why it's not making it in for el7

Reply:
But probably a good idea to double check that setting this won't break windows, when you get to working on that platform.